doc/tutorials/langevin_dynamics/langevin_dynamics.ipynb

```diff
@@ -4,78 +4,61 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>Not sure I would call the friction term in the Langevin a hydrodynamic interaction. This might give the impression that flow fields are somehow involved, especially when we later have the polymer tutorial where we do LB-hydrodynamics vs "dry" langevin</li><li>"a rapidly varying force&nbsp;f(t)&nbsp;with the time" -&gt; a force f(t) that varies rapidly in time.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/4052/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>